pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -51,6 +53,8 @@ func (p *ConfigurationEditFlags) AddUpdateFlags(command *cobra.Command) {
 func (p *ConfigurationEditFlags) AddCreateFlags(command *cobra.Command) {
 	p.AddUpdateFlags(command)
 	command.Flags().BoolVar(&p.ForceCreate, "force", false, "Create service forcefully, replaces existing service if any.")
+	command.Flags().BoolVar(&p.Async, "async", false, "Don't wait for the service to become Ready.")
+	command.Flags().IntVar(&p.WaitTimeout, "wait-timeout", 60, "Wait for that many seconds before giving up when creating a service.")
```

Comment:
"Seconds to wait before giving up on creating the service"

Reply:
The only issue is that all the other help texts are in Imperative format ("Create ..." / "Don't wait ...") and not nominal style. So should we change all to nominal style, like in

```
--force                            Forceful service creation,  replacing an existing service
--async                           Service creation in the background
--wait-timeout <secs> Seconds to wait before giving up on creating the service
```

or we keep imperative style for `--wait-timeout` but reword to e.g.

```
--wait-timeout <60> Set maximal seconds to wait when creating a service
```

" Before giving up on creating the service" is also not technically correct as the service still gets possibly created, it's just that the CLI command returns. The timeout happens most of the time if there is a large image to pull on first service creation.

Reply:
... right "before giving up on waiting for the Service to be ready"